src/graphql/schemas/typeDefs/blueprintTypeDefs.ts

```diff
@@ -0,0 +1,26 @@
+import { Field, ObjectType } from "type-graphql";
+import { GraphQLJSON } from "graphql-scalars";
+import { User } from "./userTypeDefs.js";
+
+@ObjectType()
+class Blueprint {
+  @Field()
+  id?: number;
+
+  @Field()
+  created_at?: string;
+
+  @Field(() => GraphQLJSON)
+  form_values?: typeof GraphQLJSON;
+
+  @Field()
+  minter_address?: string;
+
+  @Field()
+  minted?: boolean;
+
+  @Field(() => [User])
+  admins?: User[];
```

Comment:
I see `admin` and `admins`, which one should it be? I'm assuming an [] scales better

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/177/commits/194cb7330f09d8396b50450b7bfcb8734b104212